src/us/kbase/workspace/test/controllers/handle/HandleServiceController.java

```diff
@@ -154,14 +156,17 @@ private File createHandleServiceDeployCfg(
 		hs.add("self-url", "http://localhost:" + handleServicePort);
 		hs.add("service-port", "" + handleServicePort);
 		hs.add("service-host", "localhost");
+		URL authServiceURL = new URL(authURL.toString() + "/api/legacy/KBase/Sessions/Login");
 		hs.add("auth-service-url", authServiceURL.toString());
+		hs.add("auth-url", authURL.toString());
 		hs.add("default-shock-server", shockHost);
+		hs.add("admin-token", shockAdminToken.getToken().toString());
 		
-		hs.add("mysql-host", "127.0.0.1");
-		hs.add("mysql-port", "" + mysql.getServerPort());
-		hs.add("mysql-user", USER);
-		hs.add("mysql-pass", PWD);
-		hs.add("data-source", "dbi:mysql:" + DB);
+		hs.add("mongo-host", "127.0.0.1");
+		hs.add("mongo-port", "" + mongo.getServerPort());
+		hs.add("mongo-database", DB);
+		hs.add("mongo-collection", COLLECTION);
+		hs.add("admin-roles", handleAdminRole);
```

Comment:
Why do you need to specify the collection?

Reply:
I wanted to reserve a specific collection just for handles.
https://github.com/kbase/handle_service2/blob/develop/deploy.cfg#L18

Reply:
Why does it need to be configurable though? The HS owns the database, so it can just decide what collection it wants to use. DBs aren't (or at least shouldn't be) shared between applications.

Reply:
👍 removed collection config.